Resources/views/CRUD/base_edit.html.twig

```diff
@@ -12,7 +12,7 @@ file that was distributed with this source code.
 {% extends base_template %}
 
 {% block title %}
-    {% if admin.id(object) is not null %}
+    {% if objectId|default(admin.id(object)) is not null %}
```

Comment:
I add it only at base_edit.html.twig, it's enough? or i need add it in base_edit_form,html.twig also?